src/plugins/walletConnectToDapps/components/header/DappHeaderMenuSummary.tsx

```diff
@@ -0,0 +1,86 @@
+import { CloseIcon } from '@chakra-ui/icons'
+import { MenuGroup } from '@chakra-ui/menu'
+import { Box, HStack, MenuDivider, MenuItem, VStack } from '@chakra-ui/react'
+import dayjs from 'dayjs'
+import type { FC } from 'react'
+import { useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { MiddleEllipsis } from 'components/MiddleEllipsis/MiddleEllipsis'
+import { RawText, Text } from 'components/Text'
+import { getChainAdapterManager } from 'context/PluginProvider/chainAdapterSingleton'
+import { useEvm } from 'hooks/useEvm/useEvm'
+
+import { DappAvatar } from './DappAvatar'
+
+type Props = {
+  dapp: {
```

Comment:
pull this out to a type and export and we can use in the component below

Reply:
This is just for dummy data for now. Later I'm expecting this shape to differ, based on exactly what the WC interface looks like. It's just easier to break it down into smaller steps and use dummy data before connecting WC